src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -286,7 +286,7 @@ class DataCenterFormula private constructor(
         )
 
         private var configs: List<JiraNodeConfig> = (1..2).map { JiraNodeConfig.Builder().name("jira-node-$it").build() }
-        private var loadBalancerFormula: LoadBalancerFormula = ElasticLoadBalancerFormula()
+        private var loadBalancerFormula: LoadBalancerFormula = ApacheEc2LoadBalancerFormula()
```

Comment:
We don't need to change the default. We can offer `ApacheEc2LoadBalancerFormula` and the users can choose whether they want to use it or `ElasticLoadBalancerFormula`

Reply:
The problem is that the current default behaves in an ill way. It does not distribute the load equally.

Reply:
1. It would be nice to see that problem in a red-test-first way.
2. We can still offer the bug fix without changing the default.

Reply:
Regarding 1. it will be hard to do because the test is working based on Cookie set by load balancer (won't be working with other load balancers)

Reply:
Makes sense.
We measure load balancer fairness in a LB-agnostic way in VUs, who identify the node from the footer in the HTML UI.
We could integrate VUs or some other cookie-aware HTML-sniffing, but feel free to skip that. We did not test fairness before, so I don't want to impose the dev cost onto you.